apps/meteor/app/apps/server/communication/websockets.ts

```diff
@@ -1,7 +1,11 @@
-import { AppStatusUtils } from '@rocket.chat/apps-engine/definition/AppStatus';
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { AppStatus, AppStatusUtils } from '@rocket.chat/apps-engine/definition/AppStatus';
+import { ISetting } from '@rocket.chat/core-typings';
+import { IStreamer } from 'meteor/rocketchat:streamer';
 
 import { SystemLogger } from '../../../../server/lib/logger/system';
 import notifications from '../../../notifications/server/lib/Notifications';
+import { AppServerOrchestrator } from '../orchestrator';
 
 export const AppEvents = Object.freeze({
```

Comment:
This could be a enum

Reply:
sorry what?

Reply:
the app events could be a enum type as below:

```
export enum EAppEvents  {
	APP_ADDED = 'app/added',
	APP_REMOVED = 'app/removed',
	APP_UPDATED =  'app/updated',
	APP_STATUS_CHANGE =  'app/statusUpdate',
	APP_SETTING_UPDATED =  'app/settingUpdated',
	COMMAND_ADDED =  'command/added',
	COMMAND_DISABLED =  'command/disabled',
	COMMAND_UPDATED =  'command/updated',
	COMMAND_REMOVED =  'command/removed',
	ACTIONS_CHANGED = 'actions/changed',
};
```